### PR TITLE
1.5 Changelog Changes

### PR DESCRIPTION
**Pull request by juntai-zheng**
Auto-generated OSS Changelog + manual changes

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,58 @@
 Changelog
 =========
+1.5.0 (2019-12-19)
+-----------------------
+MLflow 1.5.0 includes several major features and improvements:
+
+New Flavors and Flavor Updates:
```
Comment (mateiz) [inline CHANGELOG.rst]: Maybe write model flavors in case people don't know what flavors are.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,58 @@
...
+
+- New support for a LightGBM flavor (#2136, @harupy)
+- New support for a XGBoost flavor (#2124, @harupy)
+- New support for a Gluon flavor and autologging (#1973, @cosmincatalin)
+- TensorFlow.Keras and Keras parameters are now logged by ``autolog()`` (#2119, @juntai-zheng)
```
Comment (mateiz) [inline CHANGELOG.rst]: This sounds like fix to auto-logging; it's not really a new model flavor or a model flavor update. Might be good to call out separately.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,5 +1,58 @@
...
+- Runs automatically created by ``mlflow.tensorflow.autolog()`` and ``mlflow.keras.autolog()`` (#2088) are now automatically ended after training and/or exporting your model. See the `docs <https://mlflow.org/docs/latest/tracking.html#automatic-logging-from-tensorflow-and-keras-experimental>`_ for more details (#2094, @juntai-zheng)
+
+More features and improvements:
+
+- [CLI] When using the ``mlflow server`` CLI command, you can now expose metrics on ``/metrics`` for Prometheus via the optional --activate-parameter argument (#2097, @t-henri)
+- [CLI] The ``mlflow ui`` CLI command now has a ``--host``/``-h`` option to specify user-input IPs to bind to (#2176, @gandroz)
+- [CLI] MLflow now supports pulling git submodules while using MLflow Projects (#2103, @badc0re)
```
Comment (mateiz) [inline CHANGELOG.rst]: Capitalize Git

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,5 +1,58 @@
...
+- Fixed ``mlflow_list_artifacts()`` (R package) (#2200, @lorenzwalthert)
+- Entrypoint commands of Kubernetes jobs are now shell-escaped (#2160, @zanitete)
+- Fixed sqlalchemy sorting that was filtering some runs (#2195, @rom1504)
+- Fixed project run conda path issue (#2147, @Zangr)
+- Fixed spark model load from model repository. (#2175, @tomasatdatabricks)
+- Stripped "dev" suffix from PySpark versions (#2137, @dbczumar)
+- Fixed note editor on the experiment page (#2054, @harupy)
+- Fixed ``models serve``, ``models predict`` CLI commands against models:/ URIs (#2067, @smurching)
+- Don't unconditionally format values as metrics in generic HtmlTableView component (#2068, @smurching)
+- Fixed remote execution from windows using posixpath (#1996, @aestene)
```
Comment (mateiz) [inline CHANGELOG.rst]: Capitalize Windows

File: CHANGELOG.rst [hunk 5]
```diff
@@ -1,5 +1,58 @@
...
+- Improves performance in run requests (#2030, #2059, @jcuquemelle)
+
+Bug fixes and documentation updates:
+
+- Fixed handling of empty directories in fs based artifact repositories. (#1891, @tomasatdatabricks)
+- Fixed ``mlflow.keras.save_model()`` usage with DBFS (#2216, @andychow-db)
+- Fixed several build issues for the docker image (#2107, @jimthompson5802)
+- Fixed ``mlflow_list_artifacts()`` (R package) (#2200, @lorenzwalthert)
+- Entrypoint commands of Kubernetes jobs are now shell-escaped (#2160, @zanitete)
+- Fixed sqlalchemy sorting that was filtering some runs (#2195, @rom1504)
```
Comment (mateiz) [inline CHANGELOG.rst]: Capitalize SQLAlchemy

File: CHANGELOG.rst [hunk 6]
```diff
@@ -1,5 +1,58 @@
...
+- [UI] The artifact image view now supports animated GIFs (#2070, @harupy)
+- Adds ability to mount volumes and specify environment variables when using mlflow with docker (#1994, @nlml)
+- Adds run context for detecting job information when using MLflow tracking APIs within Databricks Jobs. The following job types are supported: notebook jobs, Python Task jobs. (#2205, @dbczumar)
+- Improves performance in run requests (#2030, #2059, @jcuquemelle)
+
+Bug fixes and documentation updates:
+
+- Fixed handling of empty directories in fs based artifact repositories. (#1891, @tomasatdatabricks)
+- Fixed ``mlflow.keras.save_model()`` usage with DBFS (#2216, @andychow-db)
+- Fixed several build issues for the docker image (#2107, @jimthompson5802)
```
Comment (mateiz) [inline CHANGELOG.rst]: Capitalize Docker, FS and stuff like this

File: CHANGELOG.rst [hunk 7]
```diff
@@ -1,5 +1,58 @@
...
+- [CLI] MLflow now supports pulling git submodules while using MLflow Projects (#2103, @badc0re)
+- [CLI] New ``mlflow models prepare-env`` command to do any preparation necessary to initialize an environment. This allows distinguishing configuration and user errors during predict/serve time (#2040, @aarondav)
+- [API] MLflow ``log_params()`` will recognize Spark ML params as keys and will now extract only the name attribute (#2064, @tomasatdatabricks)
+- [API] Exposes ``mlflow.tracking.is_tracking_uri_set()`` (#2026, @fhoering)
+- [UI] The artifact image viewer nor displays "Loading..." when it is loading an image (#1958, @harupy)
+- [UI] The artifact image view now supports animated GIFs (#2070, @harupy)
+- Adds ability to mount volumes and specify environment variables when using mlflow with docker (#1994, @nlml)
+- Adds run context for detecting job information when using MLflow tracking APIs within Databricks Jobs. The following job types are supported: notebook jobs, Python Task jobs. (#2205, @dbczumar)
+- Improves performance in run requests (#2030, #2059, @jcuquemelle)
```
Comment (mateiz) [inline CHANGELOG.rst]: It's kind of weird that some of these have headings like [API], [CLI] etc and some don't. Can you make it consistent?

Comment (mateiz) [reply CHANGELOG.rst]: Also it's not clear what "run requests" means here.

File: CHANGELOG.rst (hunk 5 above)
Comment (dbczumar) [reply CHANGELOG.rst]: This item is a small fix for order by / filter performance optimizations introduced in 1.5. we should call out the performance optimizations as a single item and move this entry to the small fixes section

File: CHANGELOG.rst (hunk 7 above)
Comment (juntai-zheng) [reply CHANGELOG.rst]: Clarified to "searching for runs"

**Review by smurching — APPROVED** [review]
Merging to unblock the release since it looks like you've addressed @dbczumar & @mateiz's comments, let's address any additional comments in a follow-up, thanks all!

**Pull request merged**